src/main/java/sora/Storage.java

```diff
@@ -0,0 +1,131 @@
+package sora;
+
+import sora.Tasks.Deadline;
+import sora.Tasks.Event;
+import sora.Tasks.Task;
+import sora.Tasks.TaskList;
+import sora.Tasks.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Storage stores TaskList.
+ * It deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    /** File path of the saved file */
+    private final String filePath;
+
+    /**
+     * Constructs a new Storage.
+     *
+     * @param filePath File path of the saved file.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Returns String of all Task Details divided by " | ".
+     * It is a Helper Function for saveTaskList(TaskList)
+     *
+     * @param taskList TaskList.
+     * @return String of all Task Details divided by " | ".
+     */
+    private String saveTaskString(TaskList taskList) {
```

Comment:
Perhaps this could be better named? I initially interpreted this as 'saving task strings' and not 'getting the task strings to be saved'.

Reply:
Sounds solid. Will Rename.